### ISSUE TITLE
Does "MCC" mean "MFCC"

### ISSUE THREAD

**Issue by suncerock** [tts-tutorial/survey#2]
In the paper, one of the acoustic features is referred to as "mel-cepstral coefficients" (MCC). Is this the same as "mel-frequency cepstral coeffiencts" (MFCC)? If yes, I would suggest that authors mention it as the term MFCC appears more frequently than MCC.

**Comment by KunZhou9646** [comment]
No, MCC is different from MFCC. To get MFCC, we need to do frequency warping first then DCT. As for MCC, the process is in verse. Besides, the filters used to get those features are also different.  MFCC is using triangle filters and MCC is using full-band filters. They are generally two widely used spectral features for speech synthesis.
